### DISCUSSION TITLE
Select multiple elements at once

### DISCUSSION THREAD

**Discussion by Aryan1982** [Atri-Labs/atrilabs-engine#428] [Guides: General]
Is there a way to select multiple elements at once?

**Comment by cruxcode** [comment]
> Is there a way to select multiple elements at once?

Hi @Aryan1982, not currently. We understand that it's important for a good developer experience, however, most of the time there are easy workarounds whenever you need to work with multiple components.

Please explain why you felt the need for selecting multiple components at once?

**Comment by darshitac11** [comment]
@cruxcode - @Aryan1982 can select the parent component. This will automatically select all the children components.
